src/Model/BaseBlock.php

```diff
@@ -85,11 +76,9 @@ public function __construct()
 
     /**
      * NEXT_MAJOR: Add return typehint.
-     *
-     * @return string
      */
     #[\ReturnTypeWillChange]
```

Comment:
this can be removed

Reply:
Since the class is not final, isn't a BC break to add `: string` by the way ?

Reply:
hm, not sure about this one really. On php 8, string is mandatory as return type for toString, or not?